server/webapp/WEB-INF/rails.new/config/routes.rb

```diff
@@ -229,13 +229,20 @@
       end
 
       namespace :admin do
+        namespace :security do
+          resources :auth_configs, param: :auth_config_id, except: [:new, :edit, :verify_connection], constraints: {auth_config_id: ALLOW_DOTS}
+          resources :roles, param: :role_name, except: [:new, :edit], constraints: {role_name: ROLE_NAME_FORMAT}
+        end
+
+        resources :templates, param: :template_name, except: [:new, :edit], constraints: {template_name: TEMPLATE_NAME_FORMAT}
         resources :repositories, param: :repo_id, only: [:show, :index, :destroy, :create, :update], constraints: {repo_id: ALLOW_DOTS}
         resources :environments, param: :name, only: [:show, :destroy, :create, :update, :index], constraints: {:name => ENVIRONMENT_NAME_FORMAT} do
           patch on: :member, action: :patch
           put on: :member, action: :put
         end
 
         post 'encrypt', controller: :encryption, action: :encrypt_value
+        post 'internal/security/auth_configs/verify_connection' => 'security/auth_configs#verify_connection', as: :internal_verify_connection
```

Comment:
you mean `security/auth_configs#verify_connection` to `security/auth_configs#verify_connection_internal_api` ?